packages/event-handler/src/rest/errors.ts

```diff
@@ -19,12 +19,12 @@ export class ParameterValidationError extends RouteMatchingError {
   }
 }
 
-abstract class ServiceError extends Error {
+export abstract class ServiceError extends Error {
   abstract readonly statusCode: HttpStatusCode;
   abstract readonly errorType: string;
   public readonly details?: Record<string, unknown>;
 
-  constructor(
+  protected constructor(
```

Comment:
This might need to be `public`

Reply:
This is like that other one, `SeriveError` is an abstract class so it can't be instantiated.